tests/framework/common.go

```diff
@@ -234,10 +238,16 @@ nodeRegistration:
 						},
 					},
 				},
+				{
+					Role: v1alpha4.WorkerRole,
+				},
 			},
 		}
 		if Td.ClusterVersion != "" {
-			clusterConfig.Nodes[0].Image = fmt.Sprintf("kindest/node:%s", td.ClusterVersion)
+			for i := 0; i <= 3; i++ {
```

Comment:
What's 3? Maybe define a constant?

Reply:
defined a constant and I removed enabling the CRD convertor in this PR